app/models/event.rb

```diff
@@ -0,0 +1,6 @@
+class Event < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :project
+
+  validates :title, :date, presence: true
```

Comment:
Hmm, It'd be nice to have the same validation for `date` as in #49. Maybe move it to a custom validator class?

http://guides.rubyonrails.org/active_record_validations.html#performing-custom-validations